packages/sst/src/constructs/NextjsSite.ts

```diff
@@ -271,6 +278,64 @@ export class NextjsSite extends SsrSite {
     server?.addEnvironment("REVALIDATION_QUEUE_URL", queue.queueUrl);
     server?.addEnvironment("REVALIDATION_QUEUE_REGION", Stack.of(this).region);
     queue.grantSendMessages(server?.role!);
+
+    const app = this.node.root as App;
+
+    const table = new Table(this, "RevalidationTable", {
+      partitionKey: { name: "tag", type: AttributeType.STRING },
+      sortKey: { name: "path", type: AttributeType.STRING },
+      pointInTimeRecovery: true,
+      billingMode: BillingMode.PAY_PER_REQUEST,
+      tableName: app.logicalPrefixedName("next-revalidation-table"),
+    });
+
+    table.addGlobalSecondaryIndex({
+      indexName: "revalidate",
+      partitionKey: { name: "path", type: AttributeType.STRING },
+      sortKey: { name: "revalidatedAt", type: AttributeType.NUMBER },
+    });
+
+    server?.addEnvironment("CACHE_DYNAMO_TABLE", table.tableName);
+    table.grantReadWriteData(server?.role!);
+
+    const dynamodbProviderPath = path.join(this.props.path, ".open-next", "dynamodb-provider");
+
+    if(fs.existsSync(dynamodbProviderPath)) {
+      const insertFn = new CdkFunction(this, "RevalidationInsertFunction", {
+        description: "Next.js revalidation data insert",
+        handler: "index.handler",
+        code: Code.fromAsset(
+          path.join(this.props.path, ".open-next", "dynamodb-provider")
+        ),
+        runtime: Runtime.NODEJS_18_X,
+        timeout: CdkDuration.minutes(14),
```

Comment:
Any particular reason why this is 14 minutes?

Reply:
No, it's just a typo